docs/user-guide/monitoring.md

```diff
@@ -85,7 +85,7 @@ Here is a snapshot of the a Google Cloud Monitoring dashboard showing cluster-wi
 
 ## Try it out!
 
-Now that you’ve learned a bit about Heapster, feel free to try it out on your own clusters! The [Heapster repository](https://github.com/GoogleCloudPlatform/heapster) is available on GitHub. It contains detailed instructions to setup Heapster and its storage backends. Heapster runs by default on most Kubernetes clusters, so you may already have it! Feedback is always welcome. Please let us know if you run into any issues. Heapster and Kubernetes developers hang out in the [#google-containers](http://webchat.freenode.net/?channels=google-containers) IRC channel on freenode.net. You can also reach us on the [google-containers Google Groups mailing list](https://groups.google.com/forum/#!forum/google-containers).
+Now that you’ve learned a bit about Heapster, feel free to try it out on your own clusters! The [Heapster repository](https://github.com/GoogleCloudPlatform/heapster) is available on GitHub. It contains detailed instructions to setup Heapster and its storage backends. Heapster runs by default on most Kubernetes clusters, so you may already have it! Feedback is always welcome. Please let us know if you run into any issues. Heapster and Kubernetes developers hang out on [Slack](https://github.com/docs/troubleshooting.md#slack). You can also reach us on the [google-containers Google Groups mailing list](https://groups.google.com/forum/#!forum/google-containers).
```

Comment:
While you're here, heapster has moved to the kubernetes org. Please udpate the link.

google-containers@ isn't really for OSS anymore.

I suggest linking to the other suggested channels/forums in the troubleshooting doc rather than listing them all here.